_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -437,7 +437,7 @@ public static function register_endpoints() {
 				array(
 					'methods'             => WP_REST_Server::EDITABLE,
 					'callback'            => __CLASS__ . '::update_service_api_key',
-					'permission_callback' => __CLASS__ . '::edit_others_posts_check',
+					'permission_callback' => array( 'WPCOM_REST_API_V2_Endpoint_Service_API_Keys','edit_others_posts_check' ),
```

Comment:
If we know both the class name, can we reference them here literally?

```php
'permission_callback' => 'WPCOM_REST_API_V2_Endpoint_Service_API_Keys::edit_others_posts_check',
```

(Apologies if this is a silly question, my PHP is rusty.)

Reply:
It is more common to write the call back in the array() format. So I am a bit reluctant to change it. 
I didn't want to update the whole file to use the more common format. I think it could be done in a different PR.